### ISSUE TITLE
Respawn in the Twilight Forest

### ISSUE THREAD

**Issue by Emiralp26** [ModdedMinecraftClub/issue-tracker#62]
I have a bed in the overworld in my base and I have done /sethome in my base before however after I died in the Twilight forest it spawned me in the world spawn of the Twilight Forest which was filled with monsters and spawners I blocked myself in and started to mine I eventually got out than the different biomes started to kill me my deathpoint/my portal is 5000 meters away from the Twilight Forest spawn area I am stuck and I can't do anything can I get a respawn at my base please. My discord is emi#5440.

**Comment by john01dav** [comment]
You should be able to use /spawn to get to the main spawn on the server, out of the twilight forest. Also, did /home not work?

**Comment by Emiralp26** [comment]
> You should be able to use /spawn to get to the main spawn on the server, out of the twilight forest. Also, did /home not work?

I believe I have tried /home however due to being in Ironman couldn't teleport to my home, I am logging in now to try to do /spawn.

**Comment by john01dav** [comment]
Oh, I didn't notice that it was ironman. If /spawn works it's a bug. You'll need to wait until a staff member is online.

**Comment by Emiralp26** [comment]
> 

Yeah I tried both commands they don't work I am practically stuck in the Twilight Forest Dimension.

**Comment by john01dav** [comment]
You should be at spawn now.

**Comment by john01dav** [comment]
You should be able to use /spawn to get to the main spawn on the server,
out of the twilight forest. Also, did /home not work?
